packages/peregrine/lib/talons/Header/useAccountTrigger.js

```diff
@@ -32,10 +36,15 @@ export const useAccountTrigger = props => {
     const apolloClient = useApolloClient();
     const history = useHistory();
     const location = useLocation();
-    const [revokeToken] = useMutation(signOutMutation);
+    const [revokeToken] = useMutation(signOutMutation, {
+        onError: signOutMutationErrors => {
+            setFormErrors([...formErrors, signOutMutationErrors]);
```

Comment:
Similar to my other feedback - this doesn't need handling here. The error should be destructured from the mutation response object and handled in a toast.